src/libcrun/linux.c

```diff
@@ -366,7 +366,7 @@ do_remount (int targetfd, const char *target, unsigned long flags, const char *d
 {
   int ret;
   char target_buffer[64];
-  const char *real_target = target;
+ const char *real_target = target;
```

Comment:
Nit, is this a spacing issue?

Reply:
yes thanks for noticing it.  I've pushed a new version.